src/Svg.js

```diff
@@ -0,0 +1,87 @@
+import React, { Component, forwardRef } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+export class Svg extends Component {
+    promise = undefined;
+    state = { contents: null, attributes: null };
+
+    componentDidMount() {
+        this.maybeWaitForSvg();
+    }
+
+    componentDidUpdate(prevProps) {
+        if (this.props.svg !== prevProps.svg) {
+            this.maybeWaitForSvg();
+        }
+    }
+
+    componentWillUnmount() {
+        this.resetWaitForSvg();
+    }
+
+    render() {
+        const { svg, className, forwardedRef, ...rest } = this.props;
+        const { contents, attributes } = this.state;
+
+        if (contents === null) {
+            return null;
+        }
+
+        return (
+            <svg
+                { ...attributes }
+                { ...rest }
+                className={ classNames('react-svg', className) }
+                ref={ forwardedRef }
+                dangerouslySetInnerHTML={ { __html: contents } } />
+        );
+    }
+
+    parseSvg(svg) {
+        if (svg == null || typeof window === 'undefined') {
```

Comment:
I may be looking at this the wrong way, but why do we refuse the svg content if window is undefined? If we're in the SSR stage, and if the svg is inlined, it could still be rendered, no? Just trying to understand the reasoning behind this part of the condition.

Reply:
Nevermind, just noticed a few lines below `window` is being used for that 🤦 Maybe we should leave as is and avoid increasing the bundle size, then